ci/remap-scala-test-short-names.ps1

```diff
@@ -0,0 +1,68 @@
+# $ErrorActionPreference = 'Stop' causes the script to fail because Bazel writes to stderr.
```

Comment:
Doesn't this need a copyright header?

Reply:
I've added one.

Reply:
Looks like the copyright check tool doesn't look at .ps1 files.